x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/conversations/conversations.spec.ts

```diff
@@ -68,69 +68,72 @@ export default function ApiTest({ getService }: FtrProviderContext) {
-          .expect(404);
+          });
+        } catch (error) {
+          expect(error.status).to.be(404);
+        }
       });
 
       it('returns a 404 for retrieving a conversation', async () => {
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
+        try {
+          await observabilityAIAssistantAPIClient.editor({
             endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
             params: {
               path: {
                 conversationId: 'my-conversation-id',
               },
             },
-          })
-          .expect(404);
+          });
+        } catch (error) {
+          expect(error.status).to.be(404);
+        }
       });
     });
 
-    describe('when creating a conversation with the write user', function () {
+    describe('when creating a conversation with the write user', () => {
       let createResponse: Awaited<
         SupertestReturnType<'POST /internal/observability_ai_assistant/conversation'>
       >;
-
       before(async () => {
-        createResponse = await observabilityAIAssistantAPIClient
-          .slsEditor({
-            endpoint: 'POST /internal/observability_ai_assistant/conversation',
-            params: {
-              body: {
-                conversation: conversationCreate,
-              },
+        createResponse = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'POST /internal/observability_ai_assistant/conversation',
+          params: {
+            body: {
+              conversation: conversationCreate,
             },
-          })
-          .expect(200);
+          },
+        });
+        expect(createResponse.status).to.be(200);
       });
 
       after(async () => {
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
-            endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
-            params: {
-              path: {
-                conversationId: createResponse.body.conversation.id,
-              },
+        const { status } = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
+          params: {
+            path: {
+              conversationId: createResponse.body.conversation.id,
             },
-          })
-          .expect(200);
+          },
+        });
+        expect(status).to.be(200);
 
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
+        try {
+          await observabilityAIAssistantAPIClient.editor({
             endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
             params: {
               path: {
                 conversationId: createResponse.body.conversation.id,
               },
             },
-          })
-          .expect(404);
+          });
+        } catch (error) {
+          expect(error.status).to.be(404);
```

Comment:
What happens if the request returns a 200? Will the test still pass (I think it might)

Reply:
.. oh, this is in the after block. So less important to assert